recipe/meta.yaml

```diff
@@ -41,8 +41,13 @@ test:
     - pandas
   commands:
     - pip check
+    # Skip test suite on PyPy as it segfaults there
+    # xref: https://github.com/conda-forge/pandas-feedstock/issues/148
+    - python -c "import pandas; pandas.test()"  # [python_impl == "cpython"]
```

Comment:
As PyPy segfaults when running the test suite and this will likely require more debugging locally to address, simply skipping it and noting that issue for now. Also filed as issue ( https://github.com/conda-forge/pandas-feedstock/issues/148 )

Reply:
Yeah the pandas CI PyPy test build also fails usually with a segfault so this is expected